app/assets/stylesheets/components/organisations_table.scss

```diff
@@ -0,0 +1,19 @@
+.name-col {
+  width: 35%;
```

Comment:
Last time we talked about this, Emile had some feelings on not using custom styles - not sure what the best approach is if we want to tweak this table.